osprey/search_space.py

```diff
@@ -30,17 +30,18 @@ def __init__(self):
     def n_dims(self):
         return len(self.variables)
 
-    def add_jump(self, name, min, max, step):
+    def add_jump(self, name, min, max, step, type=float):
         """ An integer/float valued dimension bounded between
         range(min,max,step). Note that the right endpoint of the interval
-        includes `max`. This is a wrapper around the add_enum.
+        includes `max`. This is a wrapper around the add_enum. It assumes
+        that the jump is float but can also use ints. 
         """
         try:
```

Comment:
Yeah... if you ask for strings you're gonna get strings. Then you still get a helpful error message with the next line

```
In [9]: np.arange('1.0', '5.0')
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-9-f35afa3fc322> in <module>()
----> 1 np.arange('1.0', '5.0')

TypeError: unsupported operand type(s) for -: 'str' and 'str'
```